docs/workforce-management-framework/topics/con-wfm-common-steps.adoc

```diff
@@ -0,0 +1,25 @@
+= WFM Common Steps
```

Comment:
@jstaffor Leaving you with options how we going to call them. 
Community suggestions were:
- Out of the box
- Generic
- Common
- Build in